debian/control

```diff
@@ -0,0 +1,19 @@
+Source: souffle
+Maintainer: Nathan Keynes <nathan.keynes@oracle.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: autoconf, automake, bison, debhelper (>= 9), flex, libsqlite3-dev, openjdk-7-jdk | openjdk-8-jdk
```

Comment:
Yes, you are right, we need bison be at least 3.0, anything below that will fail. It's actually the case is OSX where bison there is quite old. I do not think that's an issue with automake, autoconf or flex. I have added the constraint for bison. In any case what configure should do (and what it does not do at the moment) is to check bison version and bail. I have some code for that in one of my branches (I was looking into brew packaging for OSX). I can port it to this Souffle repository if you like.